single_level.py

```diff
@@ -28,106 +28,47 @@ def get_note_types(resource):
 			note_types.append(note["jsonmodel_type"])
 	return note_types
 	
-def get_extent_number(resource):
-	extent_number = []
-	for extent in resource["extents"]:
-		if extent["number"]:
-			extent_number.append(extent["number"])
+def get_values(resource, array, value):
+	value_types = []
+	for item in resource[array]:
+		if item[value]:
+			value_types.append(item[value])
 		else:
-			extent_number.append("false")
-	return " , ".join(extent_number)
+			value_types.append("false")
+	return value_types
 	
-def get_language(resource):
+def get_single_value(resource, key):
 	d = resource
-	if 'language' in d:
-		return 'language'
+	if key in d:
+		return key
 	else:
 		return "false"
-
-def get_date_types(resource):
-	date_types = []
-	for date in resource["dates"]:
-		if date["date_type"]:
-			date_types.append("true")
-		else:
-			date_types.append("false")
-	return date_types
-
-def get_agent_roles(resource):
-	agent_roles = []
-	for agent in resource["linked_agents"]:
-		if agent["role"]:
-			agent_roles.append("true")
-		else:
-			agent_roles.append("false")
-	return " , ".join(agent_roles)
-	
+		
 def makeRow(resource):
 	global row
 	row = []
```

Comment:
I think you might be able to write the above two lines as one: `global row = []`